code/modules/clothing/spacesuits/rig/rig.dm

```diff
@@ -199,7 +199,7 @@
 		if(!piece) continue
 		piece.icon_state = "[initial(icon_state)]"
 		if(airtight)
-			piece.item_flags &= ~(STOPPRESSUREDAMAGE|AIRTIGHT|PHORONGUARD) //VOREStation edit

```

Comment:
PHORONGUARD does nothing here to actually make it so the rigsuit pieces are unable to be contaminated so I removed it.